requirements.txt

```diff
@@ -49,3 +49,5 @@ google-auth==1.30.0
 pandas==1.3.0
 pyarrow==4.0.1
 more-itertools==8.8.0
+kombu==5.1.0
+amqp==5.0.6
```

Comment:
librabbitmq is also supported instead of amqp and the docs mention that it has superior performance. However, we need gcc and other build stuff in the image to install it. Hence, not doing it just yet.